doc/examples/gatk/README.md

```diff
@@ -22,14 +25,25 @@ Archive:  GATK_Germline.zip
    creating: data/bams/
   inflating: data/bams/.DS_Store
   etc...
+```
+Change into the directory that contains the files we want to import
+```sh
 $ cd data/ref
 $ ls
 Icon  ref.dict  ref.fasta  ref.fasta.fai  refSDF
+```
+Next, we want to create our pachyderm repo and then instruct pachyderm to import those into our repo
+```sh
 $ pachctl create-repo reference
-$ pachctl put-file reference master -c -r -f .
+$ pachctl put-file reference master -r -f .
+```
+First milestone reached! Lets just check and make sure everything looks good
+```sh
 $ pachctl list-repo
 NAME                CREATED             SIZE
 reference           43 seconds ago      83.68MiB
+```
+```sh
 $ pachctl list-file reference master
 NAME                TYPE                SIZE
 DS_Store            file                8.004KiB
```

Comment:
I dont think this DS store file should be there. if I remember correctly, DS_store is some weird mac figment that it seems like got pushed into the repo along with the data. I dont think it'll break anything. Or maybe I'm totally wrong and that's a real part of the demo